ui/src/app/store/machine/types.ts

```diff
@@ -176,7 +176,8 @@ export type EventType = Model & {
   name: string;
 };
 
-export type Event = Model & {
+// This is named `MachineEvent` as there is already a DOM `Event` type.
+export type MachineEvent = Model & {
```

Comment:
I wonder if we should be prefixing all the machine types with `Machine` anyway, and do that with all the models.